tests/Feature/SharedSiteResolutionTest.php

```
@@ -0,0 +1,408 @@
+	public function currentSiteId(): int|string|null
+	{
+		return $this->context->currentSiteId();
+	}
+}
+
+/**
+ * Create an active site.
+ */
+function sharedResolutionSite( string $name, ?string $domain = null ): Site
+{
+	return Site::create( [
```

PHPStan (2.2.7)

[error] Call to an undefined static method ArtisanPackUI\Analytics\Models\Site::create().


(staticMethod.notFound)